.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
   - EMBER_TRY_SCENARIO=ember-release
   - EMBER_TRY_SCENARIO=ember-beta
   - EMBER_TRY_SCENARIO=ember-canary
+  - EMBER_TRY_SCENARIO=ember-default
```

Comment:
I think we should actually migrate to using ember-try's `versionCompatibility` feature eventually.